maxim/tests/test_prompts.py

```
@@ -128,6 +129,39 @@ def test_getPrompt_with_deployment_variables_Environment_prod_and_TenantId_123(
         self.assertEqual(prompt.version_id, data[env]["prodAndT123PromptVersionId"])
         self.assertEqual(len(prompt.messages), 2)
 
+    def test_getPrompt_with_deployment_variables_multiselect(
```

Ruff (0.11.9)

Missing return type annotation for public function `test_getPrompt_with_deployment_variables_multiselect`

Add return type annotation: `None`

(ANN201)